WinManifest.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+    <!-- Uniquely identifiy the application (name + version) -->
+    <assemblyIdentity
+        type="win32"
+        name="BurntSushi.github.ripgrep"
+        version="13.0.0.0"
```

Comment:
Is this required? I try not to put the version number in too many places. Right now, it's in exactly two places, and in the most recent release, I forgot to update one of them.

If it's unavoidable, then I think adding a step to `RELEASE-CHECKLIST.md` would be the right thing to do.